src/kernels/backproject.cl

```diff
@@ -64,9 +64,7 @@ backproject_tex (read_only image2d_t sinogram,
     const float by = idy - axis_pos + y_offset + 0.5f;
     float sum = 0.0f;
 
-#ifdef DEVICE_GEFORCE_GTX_TITAN_BLACK
-#pragma unroll 8
-#endif
+#pragma unroll 14
```

Comment:
So wait. You measured this on a black or a regular titan? If the latter why not specifying the unroll number in an `#else` branch?